<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerGUI</class>
 <widget class="QDialog" name="PlayerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TerraME :: Player</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btPlayPause">
       <property name="toolTip">
        <string>Execute/Pause the simulation</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="../../observerResource.qrc">
         <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btStep">
       <property name="text">
        <string>Step</string>
       </property>
       <property name="icon">
        <iconset resource="../../observerResource.qrc">
         <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btStop">
       <property name="toolTip">
        <string>Abort the execution</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../../observerResource.qrc">
         <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textMessage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btPlayPause</tabstop>
  <tabstop>btStep</tabstop>
  <tabstop>btStop</tabstop>
 </tabstops>
 <resources>
  <include location="../../observerResource.qrc"/>
 </resources>
 <connections/>
</ui>
